### PR TITLE
move from_fits to base classes

### PR DESCRIPTION
**Pull request by mirca**

### CODE REVIEW

File: lightkurve/lightcurvefile.py [hunk 1]
```diff
@@ -68,6 +68,28 @@ def cadenceno(self):
...
+            Path or URL of a FITS file.
+        **kwargs : dict
+            Keyword arguments that will be passed to the constructor.
+
+        Returns
+        -------
+        tpf : LightCurveFile object
+            The loaded light curve file.
+        """
+        return cls(path_or_url, **kwargs)
```
Comment (barentsen) [inline lightkurve/lightcurvefile.py:91]: Ohhhhh.  I didn't even know `cls` existed.  I'm such a n00b.  Wowww!

**Comment by barentsen** [comment]
Sweet!  Post a 🌯 if you want me to merge!

File: lightkurve/lightcurvefile.py (hunk 1 above)
Comment (mirca) [reply lightkurve/lightcurvefile.py:91]: Duuuude, you just forgot, I learned that with you! 😄

**Comment by mirca** [comment]
🌯🌯🌯

File: lightkurve/lightcurvefile.py (hunk 1 above)
Comment (barentsen) [reply lightkurve/lightcurvefile.py:91]: When you're not here I lose all my skills!

**Comment by barentsen** [comment]
Ok!  Merging three times!

**Pull request merged**